Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM python:3.10.6-slim
+WORKDIR /app
+RUN apt update
+RUN apt install -y wget git python3 python3-venv libgl1 libglib2.0-0
```

Comment:
The `python:3.10.6` image should already contain Python, shouldn't it?

Reply:
Thanks for the review! Yup good catch, `python3 python3-venv` is removed now